recipes/nebari/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
This can be removed, it's already included in the source dist.

Reply:
Hi @BastianZim, thanks, I just removed the License file.